openedx/core/djangoapps/theming/tests/test_theme_style_overrides.py

```diff
@@ -4,11 +4,14 @@
 import unittest
 
 from django.conf import settings
+from django.core.urlresolvers import reverse
 from django.test import TestCase, override_settings
 from django.contrib import staticfiles
 
 from openedx.core.djangoapps.theming.tests.test_util import with_comprehensive_theme
 
+from student.tests.factories import UserFactory
```

Comment:
Nitpick: this app belongs to edx-platform, so there's no need for the whitespace above this line.